backend/estimate-service/src/main/java/org/egov/works/validator/EstimateServiceValidator.java

```diff
@@ -23,12 +22,23 @@
 @Slf4j
 public class EstimateServiceValidator {
 
-    public void validateCreateEstimate(EstimateRequest request, Object mdmsData) {
+    @Autowired
+    private MDMSUtils mdmsUtils;
+
+    public void validateCreateEstimate(EstimateRequest request) {
         Map<String, String> errorMap = new HashMap<>();
         validateRequestInfo(request.getRequestInfo(), errorMap);
         validateEstimate(request.getEstimate(), errorMap);
         validateWorkFlow(request.getWorkflow(), errorMap);
+
+        String tenantId = request.getEstimate().getTenantId();
+        //split the tenantId
+        tenantId = tenantId.split("\\.")[0];
```

Comment:
Make this a new variable named like `rootTenantId`.

Reply:
shall i make it as `stateTenantId`